Sources/MapboxCoreNavigation/Router.swift

```diff
@@ -245,72 +241,89 @@ extension InternalRouter where Self: Router {
         if isRerouting { return }
         isRerouting = true
         
-        getDirections(from: location, along: routeProgress) { [weak self] (session, result) in
-            self?.isRerouting = false
-            
-            guard case let .success(response) = result else {
-                return
-            }
-            guard let route = response.routes?.first else { return }
-            
-            self?.lastProactiveRerouteDate = nil
+        calculateRoutes(from: location, along: routeProgress) { [weak self] (session, result) in
+            guard let strongSelf = self else { return }
```

Comment:
nit: `guard let self = self else { return }` but we use strongSelf as well. so just For you information.

Reply:
Thanks, fixed.